.gitmodules

```diff
@@ -12,7 +12,7 @@
 	branch = master
 [submodule "Tools/sitl_gazebo"]
 	path = Tools/sitl_gazebo
-	url = https://github.com/PX4/PX4-SITL_gazebo.git
+	url = https://github.com/Airtonomy/flight-px4-gazebo.git
```

Comment:
Just double confirm if there is any difference to use https and ssh "git@github.com:Airtonomy/flight-px4-gazebo.git"

Reply:
It doesn't seem to make a difference. Both will link up to our forked version.